tools/cmake/requirements.txt

```diff
@@ -2,3 +2,5 @@ prettytable==0.7.2
 future==0.16.0
 Jinja2>=2.10.1,<2.11
 intelhex>=2.3.0,<3.0.0
+cbor>=1.0.0
+Click>=7.0,<7.1
```

Comment:
Why set the upper bound to Click 7.1? Why not 8? Seems like we're setting ourselves up for more dependency conflicts down the line by over-specifying this. If we do require this particular range for some reason, please explain why in the commit message.

Reply:
It should be the same as in #14780: `Click>=7.0,<8`